scalafix-tests/output/src/main/scala/test/redundantSyntax/StringInterpolator.scala

```diff
@@ -0,0 +1,25 @@
+package test.redundantSyntax
+
+class StringInterpolator {
+
+  val a = 42d
+
+  var b = ""
+  b = "foo"
+  b = "foo"
+  b = s"foo $a bar"
+
+  b = """foo"""
+  b =
+    """foo
+       |bar"""
+  b = s"""foo $a bar"""
+  b = s"""$a"""
```

Comment:
If `a` is a `String`

```suggestion
  b = a
```

but i guess it's not doable with a syntactic rule